test/remote/db_tests.js

```diff
@@ -202,6 +213,8 @@ describe('remote db', function() {
           return db.deleteSessionToken(sessionToken)
         })
         .then(function() {
+          assert.equal(redisDelSpy.lastCall.args[0], account.uid)
```

Comment:
We need a bit more coverage here around the delete behaviour I think. For instance, if there is a problem with `deleteSessionToken` accidentally removing all tokens for the user, this assertion fails to spot that. If there were two tokens and it asserts the correct one is deleted, then we'd be in a stronger position.